it-tests/SwitchBetweenTabs.test.ts

```diff
@@ -0,0 +1,76 @@
+import { By, EditorView, until, VSBrowser, WebDriver, WebElement, WebView } from 'vscode-extension-tester';
+import { openAndSwitchToKaotoFrame } from './Util';
+import { join } from 'path';
+import { expect } from 'chai';
+
+describe('Switching between editor tabs', function () {
+	this.timeout(90_000)
+
+	const WORKSPACE_FOLDER = join(__dirname, '../test Fixture with speci@l chars');
+
+	const locators = {
+		EditorTabs: {
+			tabsList: 'pf-v5-c-tabs__list',
+			tabsItem: 'pf-v5-c-tabs__item',
+			activeTabAttribute: 'aria-selected',
+			tabLabelAttribute: 'aria-label',
+			tabText: 'pf-v5-c-tabs__item-text',
+			designTabLabel: 'Design canvas',
+			beansTabLabel: 'Beans editor',
+		}
+	}
+
+	let driver: WebDriver;
+	let kaotoWebview: WebView;
+
+	before(function () {
+		driver = VSBrowser.instance.driver;
+	});
+
+	after(async function () {
+		await kaotoWebview.switchBack();
+		await new EditorView().closeAllEditors();
+	});
+
+	it('Open "my.camel.yaml" file and check "Design" tab is active by default', async function () {
+		kaotoWebview = (await openAndSwitchToKaotoFrame(
+			WORKSPACE_FOLDER,
+			'my.camel.yaml',
+			driver,
+			true
+		)).kaotoWebview;
+		expect(await getTabsCount()).to.equal(2);
+		expect(await getActiveTabName()).to.equal('Design');
+	});
+
+	it('Switch to "Beans" tab and check it is active', async function () {
+		await switchToTab(locators.EditorTabs.beansTabLabel);
+		expect(await getActiveTabName()).to.equal('Beans');
```

Comment:
I think we need to check that there is some content loaded

Reply:
added wait for elements are located